lib/store/src/sortStories.ts

```diff
@@ -0,0 +1,22 @@
+import stable from 'stable';
+import { Comparator } from '@storybook/addons';
+import { storySort } from './storySort';
+
+export const sortStories = (stories: any[], storySortParameter: any, fileNameOrder: string[]) => {
```

Comment:
Can we type `stories` and `storySortParameter`?

Also I feel like the input to `sortStories` should be sensible (why are `parameters` separated from the rest of the story?) and we can do weird things to it for back-compat here.

Reply:
Done